New package: BcubeCGNS v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: BcubeCGNS
- Repository: https://github.com/bcube-project/BcubeCGNS.jl
- Created by: @bmxam
- Version: v0.1.0
- Commit: 8e39abf25caacd8b5286be4c65edc8fc1d9544c3
- Reviewed by: @bmxam
- Reference: https://github.com/bcube-project/BcubeCGNS.jl/issues/4#issuecomment-2514956598
- Description: Implementation of Bcube IO interface for CGNS format
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a835981129c6b5b2f7743355675b54c3b96ee1186f7a3de9daac4baed599449fb6d712a189e2e175e205e453d6f8ee20af5a13fdf58bb691c947d274e6d8f17079e09fd23d8fb66543e214dfd91f2d1883fecbc18ac160271eb67e76c8d9b9835012e90450a9892586df6927caaa7b0dfb4cbaca15e22de33637dcb8e38a770e00e970 -->

goerz commented:
This has the same issue with missing documentation as I mention in https://github.com/JuliaRegistries/General/pull/120345#issuecomment-2507974860

JuliaRegistrator committed:
New package: BcubeCGNS v0.1.0

UUID: a9bead8e-c4b5-4f51-8a4e-84805ae90250
Repo: https://github.com/bcube-project/BcubeCGNS.jl.git
Tree: cdac752722bd0c17ebe8d6916669e2e54d5eae22

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
